src/bijectors/corr.jl

```diff
@@ -405,17 +407,18 @@ function _inv_link_chol_lkj_rrule(y::AbstractVector)
     logJ = zero(T)
 
     z_vec = tanh.(y)
+    lc_vec = LogExpFunctions.logcosh.(y)
```

Comment:
Same here, why move this outside of the loop? Also, I'd suggest
```suggestion
    z_vec = map(tanh, y)
    lc_vec = map(LogExpFunctions.logcosh, y)
```